### DISCUSSION TITLE
Portuguese Translator's Group

### DISCUSSION THREAD

**Discussion by jlooper** [microsoft/ML-For-Beginners#374] [Translations]
hi, friends, let's discuss the translation tasks still to be done for pt-pt and pt-br

**Comment by Nockkk** [comment]
Hello @jlooper. Yes, we can split the tasks still to be done for pt-pt.
@claire-me

**Reply by clairecharles** [reply]
Heyy @Nockkk, @hi-hi-ray  and @jlooper I would be translating the following files:

- Regression base README
      - Tools README
      - Tools assignment
      - Data README
      - Data assignment
      - Linear README
      - Linear assignment
      -  Logistic README
      - Logistic assignment

**Reply by hi-hi-ray** [reply]
Sure! are you doing PT-BR? I'm Translating for PT-BR =]

**Reply by clairecharles** [reply]
Ok. I am doing pt-pt

**Reply by Nockkk** [reply]
Ok, @claire-me.
I will start translating the following files for pt-pt:

- Quizzes
- Introduction base README
  - Intro to ML README
  - Intro to ML assignment
  - History of ML README
  - History of ML assignment
  - Fairness README
  - Fairness assignment
  - Techniques of ML README
  - Techniques of ML assignment

**Comment by hi-hi-ray** [comment]
Hi! Everyone!

I'm doing the PT-BR Translation. For my first [PR](https://github.com/microsoft/ML-For-Beginners/pull/377) I'm gonna translate the following files:
- Base README.md
- Quizzes
- Introduction base README
   - Intro to ML README
   - Intro to ML assignment
   - History of ML README
   - History of ML assignment
   - Fairness README
   - Fairness assignment
   - Techniques of ML README
   - Techniques of ML assignment

If until 15 of October no one gets other chapters, I'll continue to translate chapter by chapter. Feel free to tag me so we can combine and divide the translation work. =]

**Comment by jlooper** [comment]
hi, everyone, can we standardize the vocabulary for both pt-br and pt for words like "Pre-Lecture Quiz" and "Post-Lecture Quiz" as there are several options given in the translations? Thank you all!

**Reply by hi-hi-ray** [reply]
I guess. I tried to maintain the standard in all of my translations but i guess we que put:
Pre-Lecture Quiz:  Questionário pré-aula 
Post-Lecture Quiz: Questionário pós-aula

What you all think?
